corehq/apps/receiverwrapper/views.py

```diff
@@ -89,8 +92,12 @@ def _process_form(request, domain, app_id, user_id, authenticated,
         last_sync_token=couchforms.get_last_sync_token(request),
         openrosa_headers=couchforms.get_openrosa_headers(request),
     )
-    with TimingContext() as timer:
-        result = submission_post.run()
+    try:
+        with TimingContext() as timer:
+            result = submission_post.run()
+    except ProtocolError:
```

Comment:
we're pretty sure this means riak the service is struggling and it's not an application issue?